platform/linux-generic/include/odp_packet_internal.h

```diff
@@ -96,6 +96,9 @@ typedef struct {
 	 * Members below are not initialized by packet_init()
 	 */
 
+	/* Type of extra data */
+	uint8_t extra_type;
```

Comment:
We still have 16 bits of padding between this field and `flow_hash`. Should that be explicit? I'm Ok either way, but if we're trying to pack the header having everything mapped explicitly would seem desirable.